homeassistant/components/mqtt/vacuum/schema_legacy.py

```diff
@@ -215,12 +215,18 @@ async def async_setup_entity_legacy(
 class MqttVacuum(MqttEntity, VacuumEntity):
     """Representation of a MQTT-controlled legacy vacuum."""
 
-    _default_name = DEFAULT_NAME
-    _entity_id_format = ENTITY_ID_FORMAT
+    _attr_battery_level: int | None = 0
+    _attr_is_on: bool | None = False
+    _attr_fan_speed: str | None = "unknown"
```

Comment:
These are initialized in the super class
```suggestion
```

Reply:
True but they are initialized to `None`.